### ISSUE TITLE
Dependencies are woefully out of date

### ISSUE THREAD

**Issue by pmonks** [finos/metadata-tool#25]
### Steps to Reproduce:
 1. `cd metadata-tool`
 2. `lein ancient`

### Expected Result:
No out-of-date dependencies are listed.

### Actual Result:
```
$ lein ancient
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by clojure.lang.Reflector (file:/usr/local/Cellar/leiningen/2.8.3/libexec/leiningen-2.8.3-standalone.jar) to method com.sun.xml.internal.stream.XMLInputFactoryImpl.createXMLStreamReader(java.io.Reader)
WARNING: Please consider reporting this to the maintainers of clojure.lang.Reflector
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[org.clojure/clojure "1.10.0"] is available but we use "1.9.0" (use :check-clojure to upgrade)
[org.clojure/tools.cli "0.4.1"] is available but we use "0.3.7"
[cheshire "5.8.1"] is available but we use "5.8.0"
[mount "0.1.15"] is available but we use "0.1.12"
[metosin/scjsv "0.5.0"] is available but we use "0.4.1"
[irresponsible/tentacles "0.6.3"] is available but we use "0.6.2"
[cc.qbits/spandex "0.6.4"] is available but we use "0.6.2"
[com.draines/postal "2.0.3"] is available but we use "2.0.2"
[joda-time "2.10.1"] is available but we use "2.10"
[clj-http "3.9.1"] is available but we use "3.9.0"
[midje "1.9.4"] is available but we use "1.9.1"
```

**Comment by pmonks** [comment]
This issue is getting worse; from the `dev` branch:

```
$ lein do deps, ancient
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by clojure.lang.InjectedInvoker/0x0000000800062040 to method com.sun.xml.internal.stream.XMLInputFactoryImpl.createXMLStreamReader(java.io.Reader)
WARNING: Please consider reporting this to the maintainers of clojure.lang.InjectedInvoker/0x0000000800062040
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[org.clojure/clojure "1.10.0"] is available but we use "1.9.0" (use :check-clojure to upgrade)
[org.clojure/tools.cli "0.4.2"] is available but we use "0.3.7"
[org.clojure/tools.logging "0.5.0-alpha.1"] is available but we use "0.4.1"
[org.slf4j/jcl-over-slf4j "1.7.26"] is available but we use "1.7.25"
[org.slf4j/log4j-over-slf4j "1.7.26"] is available but we use "1.7.25"
[org.slf4j/jul-to-slf4j "1.7.26"] is available but we use "1.7.25"
[cheshire "5.8.1"] is available but we use "5.8.0"
[mount "0.1.16"] is available but we use "0.1.12"
[metosin/scjsv "0.5.0"] is available but we use "0.4.1"
[irresponsible/tentacles "0.6.3"] is available but we use "0.6.2"
[cc.qbits/spandex "0.6.4"] is available but we use "0.6.2"
[com.draines/postal "2.0.3"] is available but we use "2.0.2"
[joda-time "2.10.1"] is available but we use "2.10"
[clj-http "3.10.0"] is available but we use "3.9.0"
[midje "1.9.8"] is available but we use "1.9.1"
```

**Comment by maoo** [comment]
Fixed on dev! And added `lein ancient` command in the travis build, to make it easier to spot outdated libraries.
